numpy/distutils/command/build_ext.py

```diff
@@ -566,6 +584,24 @@ def _libs_with_msvc_and_fortran(self, fcompiler, c_libraries,
                     if self.build_temp not in c_library_dirs:
                         c_library_dirs.append(self.build_temp)
 
+    def _get_msvcp_dll(self):
+        assert self.compiler.compiler_type == 'msvc'
+
+        vc_root = self.compiler.cc
```

Comment:
The compiler object doesn't have the `cc` attribute if all files were already compiled on a previous round.
So you need to add
```
if not self.compiler.initialized:
    self.compiler.initialize()
```
before that.

Reply:
Will do.